docs/dotnet/dotnet-http-metrics.md

```diff
@@ -0,0 +1,259 @@
+  * [Metric: `http.client.connection.duration`](#metric-httpclientconnectionduration)
+  * [Metric: `http.client.request.time_in_queue`](#metric-httpclientrequesttime_in_queue)
+  * [Metric: `http.client.active_requests`](#metric-httpclientactive_requests)
+- [HTTP server](#http-server)
+  * [Metric: `http.server.request.duration`](#metric-httpserverrequestduration)
+  * [Metric: `http.server.active_requests`](#metric-httpserveractive_requests)
+
+<!-- tocstop -->
+
+## HTTP client
+
+All Http client metrics are reported by `System.Net.Http` meter.
+
+### Metric: `http.client.request.duration`
+
+Client request duration measures time it takes to receive response headers and does not include time to read response body.
+
+This metric follows common [http.client.request.duration](../http/http-metrics.md#metric-httpclientrequestduration) definition.
+
+Notes:
+
+- Meter name is `System.Net.Http`
+- Metric added in .NET 8.0
+- When `error.type` attribute is reported, it contains one of [HTTP Request errors](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestError.cs), a full exception type, or a string representation of received status code.
+- The [`url.scheme`](../url/url.md) attribute is reported. <!--TODO https://github.com/open-telemetry/semantic-conventions/pull/357-->
+
+### Metric: `http.client.open_connections`
+
+<!-- semconv metric.dotnet.http.client.open_connections(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `http.client.open_connections` | UpDownCounter | `{connection}` | Number of outbound HTTP connections that are currently active or idle on the client [1] |
+
+**[1]:** Meter name: `System.Net.Http`; Added in: .NET 8.0
+<!-- endsemconv -->
+
+<!-- semconv metric.dotnet.http.client.open_connections(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `http.connection.state` | string | State of HTTP connection in the HTTP connection pool. | `active`; `idle` | Required |
+| [`network.protocol.version`](../general/attributes.md) | string | Version of the protocol specified in `network.protocol.name`. [1] | `1.1`; `2` | Recommended |
+| [`server.address`](../general/attributes.md) | string | Host identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to. [2] | `example.com` | Required |
+| [`server.port`](../general/attributes.md) | int | Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to. [3] | `80`; `8080`; `443` | Conditionally Required: [4] |
+| [`server.socket.address`](../general/attributes.md) | string | Server address of the socket connection - IP address or Unix domain socket name. [5] | `10.5.3.2` | Recommended: If different than `server.address`. |
```

Comment:
The current implementation doesn't report UDS.
We should remember to update this to `network.peer.address`.